src/api/v3/pdf_export.py

```diff
@@ -0,0 +1,70 @@
+"""
+View to return PDF document containg the requested pages.
+Single pages may be requested by url parameter, if no parameter is included all pages
+related to the current region and language will be returned.
+"""
+import logging
+
+from django.shortcuts import get_object_or_404
+from django.http import HttpResponse
+
+from cms.models import Region, Page
+from cms.views.pages.page_actions import export_pdf
+
+logger = logging.getLogger(__name__)
+
+
+def pdf_export(request, region_slug, language_code):
+    """
+    View function that either returns the requested page specified by the
+    GET parameter or returns all pages of current region and language as PDF document
+    by manipulating the GET parameter (see :class:`django.http.QueryDict`)
+    and forwarding the request to view :func:`~cms.views.pages.page_actions.export_pdf`
+
+    :param request: The request that was sent to the server
+    :type request: ~django.http.HttpRequest
+
+    :param region_slug: Slug defining the region
+    :type region_slug: str
+
+    :param language_code: The current language code
+    :type language_code: str
+
+    :return: The requested pages as PDF document (inline)
+    :rtype: ~django.http.HttpResponse
+    """
+    region = Region.get_current_region(request)
+    # since instances of GET are immutable,
+    # they must be copied before modified
+    request.GET = request.GET.copy()
+    if request.GET.get("url"):
+        # remove leading and trailing slashed to avoid ambiguos urls
+        url = request.GET.get("url").strip("/")
+        # get page candidate by filtering for translation slug
+        page = get_object_or_404(
+            Page, region=region, translations__slug=url.split("/")[-1]
+        )
```

Comment:
```suggestion
        # the last path component of the url is the page translation slug
        page_translation_slug = url.split("/")[-1]
        # get page by filtering for translation slug and translation language code
        page = get_object_or_404(
            region.pages,
            translations__slug=page_translation_slug,
            translations__language__code=language_code,
        )
```
I know you just copied+pasted the code from the single page endpoint, but it currently has a flaw: Translation slugs are only unique by region + language, so if two translations from two different languages have the same slug (this usually happens when cities duplicate e.g. the German version to other languages without really translating them), this raises the exception `cms.models.pages.page.Page.MultipleObjectsReturned: get() returned more than one Page`.